flink-core/src/main/java/org/apache/flink/util/NetUtils.java

```diff
@@ -423,7 +423,7 @@ public static Iterator<Integer> getPortRangeFromString(String rangeDefinition)
                 final int port = Integer.parseInt(range);
                 if (!isValidHostPort(port)) {
                     throw new IllegalConfigurationException(
-                            "Invalid port configuration. Port must be between 0"
+                            "Invalid port configuration. Port must be between 0 "
```

Comment:
This is a trivial change, but if we are making it , I suggest making the following change to the words as well:  
65535 is the maximum so between is not inclusive, I would say` in the range` as per the isValidHostPort javadoc.
Same for the other messages.